console/backend/pkg/model/workspace.go

```diff
@@ -0,0 +1,39 @@
+package model
+
+const WorkspacePrefix = "workspace-"
+const WorkspaceKubeDLLabel = "kubedl.io/workspace-name"
+
+// WorkspaceInfo is the object returned in http call
+type WorkspaceInfo struct {
+	Username string `json:"username"`
+
+	Namespace string `json:"namespace"`
+
+	CPU int64 `json:"cpu"`
+
+	Memory int64 `json:"memory"`
```

Comment:
what is the unit of cpu/memory/gpu value ?

Reply:
this is not implemented yet...